File: Models/Answer.cs
using System.Text.Json.Serialization;

namespace OSKIDemo.Models;

public class Answer : BaseEntity
{
    public Guid QuestionId { get; set; }
    [JsonIgnore]
    public Question Question { get; set; }
    public string Text { get; set; }
    public bool IsRight { get; set; }
}

File: client/src/app/models/answer.ts
import { Guid } from 'guid-typescript';

export default class Answer {
  id: Guid;
  text: string;
  isRight: boolean;

  constructor(answer: Answer) {
    this.id = answer.id;
    this.text = answer.text;
    this.isRight = answer.isRight;
  }
}
